addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -256,8 +318,7 @@ private void checkWildcardSources(
                             "pscanrules.csp.otherinfo.extended",
                             String.join(", ", allowedDirectivesWithoutFallback));
         }
-        buildWildcardAlert(getHeaderField(msg, HTTP_HEADER_CSP).get(0), csp, wildcardSrcOtherInfo)
```

Comment:
Same comment regarding the header no longer matching the case (I know this is now used for different places but still). I guess the isMeta could be used for that.